lxqt-config-brightness/main.cpp

```diff
@@ -54,16 +54,17 @@ int main(int argn, char* argv[])
     if( parser.isSet(increaseOption) || parser.isSet(decreaseOption) || parser.isSet(setOption) )
     {
         XRandrBrightness *brightness = new XRandrBrightness();
-        QList<MonitorInfo> monitors = brightness->getMonitorsInfo();
+        const QList<MonitorInfo> monitors = brightness->getMonitorsInfo();
         QList<MonitorInfo> monitorsChanged;
         float sign = parser.isSet(decreaseOption)?-1.0:1.0;
         double brightness_value = parser.value(setOption).toFloat();
         brightness_value = qMin( qMax(brightness_value, 0.0), 100.0 ) / 100.0;
         if(!parser.value(setOption).isEmpty())
             sign = 0.0;
-        foreach(MonitorInfo monitor, monitors)
+        for (auto i = monitors.cbegin(); i != monitors.cend(); ++i)
```

Comment:
Just curious... why aren't you using the range-for loop here?

Reply:
It's an artifact. An experiment to trigger a container detachment.
Actually I inadvertently pushed lxqt-config without a final revision.